### PR TITLE
Cluster Serving flink distributed to remote and load local model

### PR DESCRIPTION
**Pull request by Litchilitchy**
Whether OpenVINO model could be broadcast successfully does not matter now.

### CODE REVIEW

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala [hunk 1]
```diff
@@ -47,7 +47,7 @@ object ClusterServing {
     helper.initArgs()
     params = new SerParams(helper)
     val serving = StreamExecutionEnvironment.getExecutionEnvironment
-
+    serving.registerCachedFile(params.modelDir, "cluster-serving-model")
```
Comment (glorysdj) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala:50]: put cluster-serving-model to your Conventions?

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/engine/FlinkInference.scala [hunk 2]
```diff
@@ -20,22 +20,26 @@ import com.intel.analytics.bigdl.tensor.Tensor
...
+
   var t: Tensor[Float] = null
   var logger: Logger = null
   var inferenceCnt: Int = 0
   var pre: PreProcessing = null
   var post: PostProcessing = null
 
   override def open(parameters: Configuration): Unit = {
+    val myDir = getRuntimeContext.getDistributedCache.getFile("cluster-serving-model").getPath
+    println(s"Model is distributed to executor path ${myDir}")
```
Comment (glorysdj) [inline zoo/src/main/scala/com/intel/analytics/zoo/serving/engine/FlinkInference.scala:41]: change println to logger debug?

**Comment by glorysdj** [comment]
need to test on cluster mode

File: zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala (hunk 1 above)
Comment (Litchilitchy) [reply zoo/src/main/scala/com/intel/analytics/zoo/serving/ClusterServing.scala:50]: noted

**Comment by Litchilitchy** [comment]
@glorysdj test passed

**Pull request merged**